sargo.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
 
-  <include name="base-q.xml"/>
+  <include name="gsi.xml"/>
```

Comment:
@lissyx I want to test if use GSI branch to build non-GSI sargo. 
If it feasible, I will be able to combine GSI and non-GSI.
Can you have time to rebuild again? 

note : 
Possibly damage the source tree. (`repo sync` to  rollback if damage).
`GITREPO=https://github.com/b2g-gsi/manifests BRANCH="ittat-patch-gsi" REPO_INIT_FLAGS="--depth=1" ./config.sh sargo`
`export SKIP_ABI_CHECKS=true` and apply patch: `./patcher/patcher.sh` before ./build.sh